Selenium_Basics/EpamNavigationTests2.cs

```diff
@@ -0,0 +1,109 @@
+        bool Americas = driver.FindElement(By.XPath("//a[@data-item='0']")).Displayed;
+        bool EMEA = driver.FindElement(By.XPath("//a[@data-item='1']")).Displayed;
+        bool APAC = driver.FindElement(By.XPath("//a[@data-item='2']")).Displayed;
+        
+        Assert.True(Americas, "not displayed");
+        Assert.True(EMEA, "not displayed");
+        Assert.True(APAC, "not displayed");
+    }
+    
+    [Test]
+    public void CheckThatUrlToSearchContainsMyText()
+    {
+        var epamUrl = "https://www.epam.com/";
+        var searchUrl = "https://www.epam.com/search?q=Automation";
+        var keyWord = "Automation";
+            
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//div[@class='header-search-ui header-search-ui-23 header__control']")).Click();
+        Thread.Sleep(1000);
+        driver.FindElement(By.XPath("//input[@id='new_form_search']")).Click();
+        driver.FindElement(By.XPath("//input[@id='new_form_search']")).SendKeys(keyWord);
+        driver.FindElement(By.XPath("//span[@class='bth-text-layer']")).Click();
+        
+        Assert.That(driver.Url,Is.EqualTo(searchUrl), "The url is wrong");
+
+        var ListOfArticles = driver.FindElements(By.XPath("//article")).ToList().Take(5);
+        var ArticlesToLower = ListOfArticles.Select(article=>article.Text.ToLower());
+        string Output = string.Join(",", ArticlesToLower);
+        foreach (var article in ArticlesToLower)
+        {
+            var a = article.Contains(keyWord.ToLower());
+        }
+        Assert.That(ArticlesToLower.All(article => article.Contains(keyWord.ToLower())),Is.True, $"Search criteria is not met: {Output}");
+    }
+    
+    [Test]
+    public void CheckThatTitleOfTheOpenedPageEqualsSearchTitle()
+    {
+        var epamUrl = "https://www.epam.com/";
+        var searchUrl = "https://www.epam.com/search?q=Business+Analysis";
+        var keyWord = "Business Analysis";
+            
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//div[@class='header-search-ui header-search-ui-23 header__control']")).Click();
+        Thread.Sleep(2000);
+        driver.FindElement(By.XPath("//input[@id='new_form_search']")).Click();
+        driver.FindElement(By.XPath("//input[@id='new_form_search']")).SendKeys(keyWord);
+        driver.FindElement(By.XPath("//span[@class='bth-text-layer']")).Click();
+        
+        Assert.That(driver.Url,Is.EqualTo(searchUrl), "The url is wrong");
+        
+        var firstArticleTitle = driver.FindElement(By.XPath("//a[@class='search-results__title-link'][1]")).Text;
+        driver.FindElement(By.XPath("//button[@id='onetrust-accept-btn-handler']")).Click();
+        Thread.Sleep(1000);
+        driver.FindElement(By.XPath("//a[@class='search-results__title-link'][1]")).Click();
```

Comment:
Minor comment to make your code more readable. Plase find element first, name it and then do actions.
ex: var searchButton = _driver.FindElement()
searchButton.Click()